server/registry.go

```diff
@@ -39,16 +41,17 @@ type (
 	}
 
 	handlerDTO struct {
```

Comment:
Does this change the API contract? i.e. does the json payload or POST body change here? Is it backwards compatible?

Reply:
it does not. the handlerDTO is not exposed to general public - it is package private method. 
follows "closed for modifications open for extensions" principle